src/fileiter.cpp

```diff
@@ -53,6 +53,12 @@ namespace std{
 #include <sys/cygwin.h>
 #endif
 
+#if defined (BOOST_ASIO_WINDOWS_RUNTIME) || defined (BOOST_PLAT_WINDOWS_PHONE)
```

Comment:
Good catch.  Indeed, the line should read:

#if defined (BOOST_**PLAT**_WINDOWS_RUNTIME) || defined (BOOST_PLAT_WINDOWS_PHONE)

(Note the use of 'PLAT' rather than 'ASIO'.) This must have leaked through from my port of ASIO and submitted to Chris Kohlhoff. :-)

I'd like to make the change to the #ifdef statement.  I'm relatively new to Git: how can I make that change?  Do I need to submit a new pull request?